client/shopping-app-clone/src/components/common/button/Button.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, useEffect, useState } from "react";
+import "assets/scss/common/button/button.scss";
+import "https://kit.fontawesome.com/637b14841d.js";
```

Comment:
why do we need this, we have already imported font-awesome globally in index.html ?

Reply:
yeah removed